recipes/keygen-py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "keygen-py" %}
+{% set version = "0.0.1.dev4" %}
```

Comment:
You need to release before we can merge. Every build on conda-forge is considered a release to end-users.